### PR TITLE
*subscription pallet testing guide

### PR DESCRIPTION
**Pull request by okalenyk**

### CODE REVIEW

File: docs/SubscriptionTestingGuide.md [hunk 1]
```diff
@@ -0,0 +1,525 @@
...
+daoId: 0
+```
+
+- For the proposal call use the following:
+
+```
+proposal: daoTreasury --> transferToken
+daoId: 0
+amount: 1000
+beneficiary: select the wallet account you'be recently created
```
Comment (gfox1) [inline docs/SubscriptionTestingGuide.md:127]: Update to "you've"

File: docs/SubscriptionTestingGuide.md [hunk 2]
```diff
@@ -0,0 +1,525 @@
...
+- Open `Developer --> Extrinsics` section
+
+- From the list of available extrinsics select `daoCouncil --> vote` callable
+
+- Use the following parameters for the voting call:
+
+```
+account: Alice
+daoId: 0
+proposal: 0x5ed15c6ac27c4bd22a5f084e4be38af2904f2e669abf5b35e7f97975aeefcbf0
```
Comment (gfox1) [inline docs/SubscriptionTestingGuide.md:174]: Add in note: Proposal hash will be different for user.

File: docs/SubscriptionTestingGuide.md [hunk 3]
```diff
@@ -0,0 +1,525 @@
...
+- repeat the voting step above using the 1st account you've created earlier
+
+- Select `daoCouncil --> close` callable
+
+- Use the following parameters for the voting close call:
+
+```
+account: 1st account from the wallet
+daoId: 0
+proposal: 0x5ed15c6ac27c4bd22a5f084e4be38af2904f2e669abf5b35e7f97975aeefcbf0
```
Comment (gfox1) [inline docs/SubscriptionTestingGuide.md:192]: Same thing here. Add note proposal hash will be different for user.

File: docs/SubscriptionTestingGuide.md [hunk 4]
```diff
@@ -0,0 +1,525 @@
...
+
+<img src="images/subscription/Screenshot%202023-07-04%20at%2010.29.36.png" width="600" style="padding-left: 50px;">
+
+## Create Proposal
+
+- Open `Developer --> Extrinsics` section
+
+- From the list of available extrinsics select `daoCouncil --> propose` callable
+
+- Use the account that you've created recently(0 SCTL tokens) as a tx submitter
```
Comment (gfox1) [inline docs/SubscriptionTestingGuide.md:471]: "you've created **first**"

File: docs/SubscriptionTestingGuide.md [hunk 5]
```diff
@@ -0,0 +1,525 @@
...
+
+## Vote for Extension Proposal
+
+- Use the same approach from the [Vote for Council Proposal](./SubscriptionTestingGuide.md/#vote-for-council-proposal) step changing the following params:
+
+```
+dao_id: 0
+proposalIndex: 1
+proposalHash: 0xbca84298f0b0c6823ef4863168bc5c545dc7553560850987b455256dc3dc950e
+```
```
Comment (gfox1) [inline docs/SubscriptionTestingGuide.md:499]: Add in:   "approve: Yes"

File: docs/SubscriptionTestingGuide.md (hunk 1 above)
Comment (okalenyk) [reply docs/SubscriptionTestingGuide.md]: done.

File: docs/SubscriptionTestingGuide.md (hunk 2 above)
Comment (okalenyk) [reply docs/SubscriptionTestingGuide.md]: replaced with generic `<proposal hash from the steps above>`

File: docs/SubscriptionTestingGuide.md (hunk 3 above)
Comment (okalenyk) [reply docs/SubscriptionTestingGuide.md]: same as above

File: docs/SubscriptionTestingGuide.md (hunk 4 above)
Comment (okalenyk) [reply docs/SubscriptionTestingGuide.md]: done.

**Review by gfox1 — APPROVED** [review]

**Pull request merged**